src/test/ui/command-argv0.rs

```diff
@@ -0,0 +1,34 @@
+// run-pass
+
+#![allow(stable_features)]
```

Comment:
Why?

Reply:
I used `src/test/ui/command-exec.rs` as a prototype and just copied it from there. I wasn't sure why.